packages/sdk/src/modules/middlewareModule/utils/getRequestSender.ts

```diff
@@ -86,6 +87,32 @@ export const getRequestSender = (options: Options): RequestSender => {
     };
   };
 
+  const defaultLogger: Logger = {
+    request: ({ config, url, params }) => {
+      const { pathname } = new URL(url);
+      // eslint-disable-next-line no-console
+      console.log(
+        `${config.method} ${pathname}`,
+        `(${typeof window === "undefined" ? "server" : "client"} side)`,
+        JSON.stringify(params)
+      );
+    },
+    response: ({ config, url, responseTime }) => {
+      const { pathname } = new URL(url);
+      // eslint-disable-next-line no-console
+      console.log(`${config.method} ${pathname} in ${responseTime}ms`);
+    },
+  };
+
+  const getLogger = (logger: Options["logger"]): Logger => {
+    if (logger === true) return defaultLogger;
+    if (logger === false) return {};
+    if (!logger)
+      return process.env.ALOKAI_SDK_DEBUG === "true" ? defaultLogger : {};
```

Comment:
This env will not be transformed into the value during the SDK build? It's was often the issue on the storefronts ;)

Reply:
hmm you're right, but this is the limitation of the frameworks... so idk if should I leave it here or remove, because in Next.js or Nuxt we'll have to specify the `logger` value anyway...

![CleanShot 2024-07-29 at 08 37 27@2x](https://github.com/user-attachments/assets/b102bc79-ec56-44ea-8060-5a6605771f3f)

Reply:
So, it stays - great!